services/QuillLMS/engines/evidence/spec/rails_helper.rb

```diff
@@ -3,6 +3,9 @@
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
 
+require 'dotenv'
+Dotenv.load('.env.test')
+
```

Comment:
We want the .env.test variables to load before config/initializers does otherwise ENV variables used in those files won't be found.